selftests/functional/test_basic.py

```diff
@@ -50,6 +50,17 @@ def test(self):
         self.log.info(hello())
 '''
 
+INVALID_PYTHON_TEST = '''
+from avocado import Test
+
+class MyTest(Test):
+
+    asdasdasd
```

Comment:
generally yes, but something like `this_is_non_existing_variable_causing_crash` might be easier to read in a few years.